index.d.ts

```diff
@@ -269,7 +269,7 @@ declare namespace RNRF {
     jump(props: props): void,
     refresh(props: props): void,
     focus(props: props): void,
-    create(scene: Scene, wrapBy?: () => any): Object
+    create(scene: React.ReactNode, wrapBy?: () => any): Object
```

Comment:
Actually, I want to define
```
create(scene: Scene, wrapBy?: () => any): Object
```
But, using this definition, I cannot write the following code.
```
Actions.create(
  <Scene key={"some_key"} />
);
```

Are there better way?